docs/source/2019HaloweenStorm.rst

```diff
@@ -110,40 +110,39 @@ Surface Temperature and Wind Speed
 .. tabs::
   .. group-tab:: MRW.v1.0
 
-    .. figure:: images/2019Halloween/2mT_MRW_v1.0_2019HalloweenStorm_trim.png
+    .. figure:: images/2019Halloween/2mT_MRW_v1.0_RAP_2019HalloweenStorm_trim.png
       :width: 1200
       :align: center
 
       2-m temperature (F) valid at 00z 1 Nov 2019 
 
-    * Colder 2-m T in MRW_GFSv15p2 along the U.S. east coast compared with GFS_ANL
-    * Colder 2-m T at New England and warmer 2-m T at the Southeast in MRW_GFSv16beta  
+    * Colder 2-m T in MRW_GFSv15p2 along the U.S. east coast compared with RAP_ANL.
+    * Colder 2-m T at New England and warmer 2-m T at the Southeast in MRW_GFSv16beta.
 
-    .. figure:: images/2019Halloween/GUST_MRW_v1.0_2019HalloweenStorm_trim.png
+    .. figure:: images/2019Halloween/GUST_MRW_v1.0_2019HalloweenStorm_RAP_trim.png
```

Comment:
Why is RAP after HalloweenStorm here, and before above in line 113?

Reply:
Good catch Dom! Comments are addressed. Thanks!